apps/fabric-website/src/components/Nav/Nav.CollapsibleSectionTitle.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { createComponent } from '@uifabric/experiments/lib/Foundation';
+import { CollapsibleSectionTitleView as view } from '@uifabric/experiments/lib/components/CollapsibleSection';
+import { getStyles as styles } from './Nav.CollapsibleSectionTitle.styles';
+import {
+  ICollapsibleSectionTitleProps,
+  ICollapsibleSectionTitleStyles
+} from '@uifabric/experiments/src/CollapsibleSection';
+
+export const CollapsibleSectionTitle: React.StatelessComponent = createComponent<
```

Comment:
`titleProps` should have a `styles` prop but it doesn't, which requires you to define a whole separate component here just to do styling. :(

Reply:
(I will fix this soon.)

Reply:
Fixed in #5693

Reply:
With the merging of #5693, Title styles is exposed in titleProps you should be able to remove this file entirely